### PR TITLE
#33 retrieve all articles

### PR DESCRIPTION
**Pull request by Banovvv**
#33

### CODE REVIEW

**Review by NataliiaBidiuk — CHANGES_REQUESTED** [review]
Please, verify that all changes are correct

File: SportsHub/SportsHub.Api/Controllers/RegisterController.cs [hunk 1]
```diff
@@ -0,0 +1,33 @@
+﻿using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using SportsHub.AppService.Authentication;
+using SportsHub.AppService.Authentication.Models.DTOs;
+
+namespace SportsHub.Api.Controllers
+{
```
[the remark this answers is not in the document]
Comment (Banovvv) [reply SportsHub/SportsHub.Api/Controllers/RegisterController.cs:7]: This was merged from `main` and not done by me, but I will check it.

[the remark this answers is not in the document]
Comment (NataliiaBidiuk) [reply SportsHub/SportsHub.Api/Controllers/RegisterController.cs:7]: check it, please. it should not be here

**Review by MBocharskyi — APPROVED** [review]

**Review by NataliiaBidiuk — APPROVED** [review]

**Comment by NataliiaBidiuk** [comment]
Add Unit tests, please

**Review by nakometa — APPROVED** [review]

**Comment by Banovvv** [comment]
> > Add Unit tests, please
> 
> I started adding those, fixed compile errors in the tests from `main`, but the test do not run, they throw no errors - just don't run:
> 
> ```
> ========== Starting test run ==========
> ========== Test run finished: 0 Tests (0 Passed, 0 Failed, 0 Skipped) run in < 1 ms ==========
> ```

This was sorted by installing `xunit.runner.visualstudio` package

**Review by MBocharskyi — APPROVED** [review]

File: SportsHub/UnitTests/Controllers/ArticleControllerTests.cs [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+            //Act
+            var result = await _articleController.GetArticleByTitleAsync(title);
+
+            //Assert
+            Assert.IsType<BadRequestObjectResult>(result.Result);
+        }
+
+        [Theory]
+        [InlineData(3)]
```
Comment (NataliiaBidiuk) [inline SportsHub/UnitTests/Controllers/ArticleControllerTests.cs:63]: InlineData is good for using when you have more than 1 parameters. use regular Fact there

Comment (Banovvv) [reply SportsHub/UnitTests/Controllers/ArticleControllerTests.cs:63]: Done

**Pull request merged**